### PR TITLE
[ah_var_store] VCF max alt alleles [VS-1334] [VS-1357]

### PR DESCRIPTION
**Pull request by mcovarr**
`ah_var_store` edition: Allows hard-filtering based on a maximum number of alt alleles [VS-1334], as well as fixing GATK Docker image building to use image IDs rather than git hashes [VS-1357].

Integration test _mostly_ successful [here](https://app.terra.bio/#workspaces/gvs-dev/GVS%20Integration/job_history/5d021859-5971-4fd7-8451-086c224fdb00).

`GvsQuickstartIntegration` failed with:

```
The bytes observed (89733530) for 'ExtractFilterTask.GvsCreateFilterSet.BigQuery Query Scanned' differ from those expected (85119360)
FAIL!!! The relative difference between these is 0.0514208, which is greater than the allowed tolerance (0.05)
```

Successful tieout run [here](https://app.terra.bio/#workspaces/gvs-dev/GVS%20Integration/job_history/04b840f9-9779-48d6-8faa-4425d67ddadb).

[VS-1334]: https://broadworkbench.atlassian.net/browse/VS-1334?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[VS-1357]: https://broadworkbench.atlassian.net/browse/VS-1357?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: scripts/variantstore/wdl/GvsTieoutVcfMaxAltAlleles.wdl [hunk 1]
```diff
@@ -0,0 +1,98 @@
+version 1.0
+
+import "GvsUtils.wdl" as Utils
+
+workflow GvsTieoutVcfMaxAltAlleles {
```
Comment (gbggrant) [inline scripts/variantstore/wdl/GvsTieoutVcfMaxAltAlleles.wdl:5]: Would it make sense to add this to a test case? Maybe run the workflow with maximum_alt_alleles defined as some largeish number?

File: src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohort.java [hunk 2]
```diff
@@ -414,6 +421,10 @@ protected void onStartup() {
                     "if no sample file (--sample-file) is provided.");
         }
 
+        OptionalLong optionalMaximumAlternateAlleles =
```
Comment (gbggrant) [inline src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohort.java:424]: I'm not opposed to OptionalLong, but it seems out of character with respect to the rest of the code which is firmly in classic Java camp.

**Review by gbggrant — APPROVED** [review]

File: src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohort.java (hunk 2 above)
Comment (mcovarr) [reply src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohort.java:424]: Hmm not sure what "classic Java" is, but`Optional`s are all over GATK. 🙂 I can make this an `Optional<Long>` if that seems less jarring.

**Review by koncheto-broad — APPROVED** [review]
LGTM.  OptionalLong is getting a little spicy for this old head, but it looks fine.  I think there's some question as to whether we could toss these sites before or after the step that merges the VariantContexts together, but I think that can be looked into as a potential change if we see failures

**Review by rsasch — COMMENTED** [review]
Seems a little misleading to have `print_debug_information` be a top-level (e.g. threaded all the way up to GvsJointVariantCalling.wdl which the Beta users use) input when it only effects VCF extraction.

**Pull request merged**